docs/DeveloperGuide.adoc

```diff
@@ -837,33 +902,369 @@ Use case ends.
 +
 Use case ends.
 
+[none]
 * 3a. The given index is invalid.
 +
 [none]
-** 3a1. AddressBook shows an error message.
+** 3a1. ContactSails shows an error message.
++
+Use case resumes at step 2.
+
+[discrete]
+=== Use case: Add person
+
+*MSS*
+
+1.	User requests to add specific person
+2.	ContactSails adds the person
++
+Use case ends.
+
+*Extensions*
+
+[none]
+* 1a. User enters invalid format for the Add feature
++
+[none]
+** 1a1. ContactSails shows an error message
++
+Use case ends.
+
+[discrete]
+=== Use case: List current customers
+
+*MSS*
+
+1.  User requests for current customers
+2.  ContactSails finds and displays a list of the list of customers whose orders are currently being fulfilled by user
```

Comment:
There is a bit of disconnect between 1 and 2. You are requesting the current customers, but the system returns customers whose orders are being fulfilled?